test/testdrive/test-skip-if.td

```diff
@@ -0,0 +1,22 @@
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ skip-if
+SELECT mz_version() LIKE '%gus%';
+
+# false skip result means we run this and see the failure
+! SELECT nonsense;
+exact:column "nonsense" does not exist
+
+$ skip-if
+SELECT mz_version() LIKE '%dev%';
```

Comment:
This won't be true on release tag builds! Honestly I might just do `SELECT false` and `SELECT true`.